docs/_guides/working-with-events.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+---
+
+# Working with Events in Dispatch
+
+Dispatch is designed to accept events from different sources. To be able to do that, Dispatch introduces a concept of event driver.
+Event Driver an image that knows how to ingest events from particular source. For example, if you would like to run your functions
```

Comment:
Just take the edits from here https://github.com/vmware/dispatch/pull/244/files#diff-3c7f82c0529b4f417e37fcaa6b9c0706R8 and replace the first line :)